src/engine/source/api/src/catalog/catalog.cpp

```diff
@@ -221,7 +221,7 @@ std::optional<base::Error> Catalog::postResource(const Resource& collection,
         if (collection.m_name.parts()[i] != contentName.parts()[i])
         {
             return base::Error {
-                fmt::format("Invalid content name \"{}\" for collection \"{}\"",
+                fmt::format(R"(Invalid content name "{}" for collection "{}")",
```

Comment:
[Coding style, Modern language features](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices)